tests/integration/conftest.py

```diff
@@ -352,9 +373,32 @@ def is_mysql_ready():
 
         await ops_test.model.block_until(is_mysql_ready, timeout=300, wait_period=3)
 
+    return wait_mysql_pod_ready
+
+
+@pytest_asyncio.fixture(scope="module", name="build_and_deploy")
+async def build_and_deploy_fixture(
+    num_units,
+    ops_test: pytest_operator.plugin.OpsTest,
+    application_name,
+    deploy_and_wait_for_mysql_pod,
+):
+    """Deploy all required charms and kubernetes pods for tests."""
+    assert ops_test.model
+
+    async def build_and_deploy_wordpress():
+        my_charm = await ops_test.build_charm(".")
+        await ops_test.model.deploy(
+            my_charm,
+            resources={"wordpress-image": "localhost:32000/wordpress:test"},
```

Comment:
It'd be good to parametrize this test so that the image can be passed as cmd arg. This would avoid coupling the test to the registry